examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -79,13 +79,15 @@ const SUPPORTS_PERMIT_ENTRYPOINTS: [EntrypointName; 2] =
 pub const NONCE_EVENT_TAG: u8 = u8::MAX - 5;
 
 /// Tagged events to be serialized for the event log.
-#[derive(Debug, Serial)]
+#[derive(Debug, Serial, Deserial, PartialEq, Eq)]
 #[concordium(repr(u8))]
 pub enum Event {
     /// The event tracks the nonce used by the signer of the `PermitMessage`
     /// whenever the `permit` function is invoked.
     #[concordium(tag = 250)]
     Nonce(NonceEvent),
+    #[concordium(forward = cis2_events)]
```

Comment:
So this changes the schema which seems like it should be a separate change since it changes the contract API that affects the dapps.

Reply:
It appends the schema so I thought it would be fine to include (no breaking changes).

Testing transactions that emit the `Nonce` event as well as cis2 events are now easier done in the test cases.

Reply:
Ok.